1-raw-data/gcs/gcs.go

```diff
@@ -93,8 +95,32 @@ func (gcs *StorageConnection) CreateBucket(ctx context.Context) (err error) {
 	}
 }
 
-// Store files in storage.
-func (gcs *StorageConnection) StoreInBucket(ctx context.Context, fileName, url string) (err error) {
+// Store url content in storage.
+func (gcs *StorageConnection) StoreTextContentInBucket(ctx context.Context, fileName, text string) (err error) {
+	//TODO add more filename validation
+	if fileName == "" {
+		return fmt.Errorf("Filename is empty.")
+	}
+
+	obj := gcs.bucket.Object(fileName)
+
+	// w implements io.Writer.
+	w := obj.NewWriter(ctx)
+
+	// Copy file into storage
+	_, err = io.Copy(w, strings.NewReader(text))
+	if err != nil {
+		log.Printf("Failed to copy %v to bucket with the error: %v", fileName, err)
```

Comment:
Should this be logged or returned?

Reply:
Using this to log something failed to write but it shouldn't stop the program from running. Sometimes we get corrupted files in the larger group and we want it to keep loading everything it can but log what was corrupted.